api/Makefile

```
@@ -1,4 +1,14 @@
-.PHONY: proto clean test build run deps dev migrate-up migrate-down docs
+.PHONY: proto sqlc clean test build run deps dev migrate-up migrate-down docs
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)